sys/net/network_layer/sixlowpan/sixlowpan_print.c

```diff
@@ -40,9 +39,9 @@ void sixlowpan_print(uint8_t *data, size_t size)
         sixlowpan_frag_t *hdr = (sixlowpan_frag_t *)data;
 
         puts("Fragmentation Header (first)");
-        printf("datagram size: %" PRIu16 "\n",
-               (uint16_t) (byteorder_ntohs(hdr->disp_size) & SIXLOWPAN_FRAG_SIZE_MASK));
-        printf("tag: 0x%" PRIu16 "\n", byteorder_ntohs(hdr->tag));
+        printf("datagram size: %u\n",
```

Comment:
I'm not sure dropping the inttypes defines is a good idea.
There were only problems with PRIu8, right?

Maybe instead of making our code non-C-standard because of misbehaviour of newlib nano, we should try to fix or work around the problem. Maybe we can inject our own inttypes.h for newlib-nano builds...

Reply:
I'm not sure either, but the output formatting is broken in this function anyways (`0x` prefix for decimal numbers?)

Reply:
Where is this issue referenced with inttypes.h macros ?

Reply:
#1891.